pages/_app.tsx

```diff
@@ -28,6 +27,11 @@ import ProjectPropsProvider, {
 import UserPropsProvider from '../src/features/common/Layout/UserPropsContext';
 import PlayButton from '../src/features/common/LandingVideo/PlayButton';
 import ErrorHandlingProvider from '../src/features/common/Layout/ErrorHandlingContext';
+import dynamic from 'next/dynamic';
+
+const VideoContainer = dynamic(() => import('../src/features/common/LandingVideo'), {
+  ssr: false,
+});
```

Comment:
If this intended to avoid loading the video, it did not work for me - does it work for you (at least the video is loaded from the cache if it was loaded before):

<img width="1596" alt="Bildschirmfoto 2022-01-13 um 17 29 51" src="https://user-images.githubusercontent.com/1532418/149369823-b86b3c69-505c-43a8-8a40-efc69a876542.png">